### PR TITLE
Need to export TreeListComponent

### PR DESCRIPTION
**Pull request by dlabrecq**
Required changes for treelist to run in planner UI, including export all treelist objects and convert styles to string.

### CODE REVIEW

**Comment by pmuir** [comment]
[test]

File: src/app/treelist/treelist.component.ts [hunk 1]
```diff
@@ -1,11 +1,11 @@
 import { Component, Input, TemplateRef, ViewChild, ViewEncapsulation } from '@angular/core';
-import { TreeNode, TREE_ACTIONS, KEYS, IActionMapping, TreeComponent } from 'angular2-tree-component';
+import { TreeNode, TREE_ACTIONS, IActionMapping, TreeComponent } from 'angular2-tree-component';
 
 // See docs: https://angular2-tree.readme.io/docs
 @Component({
   selector: 'alm-tree-list',
   template: require('./treelist.component.html'),
-  styles: [require('./treelist.component.css')],
+  styles: [require('./treelist.component.css').toString()],
```
Comment (joshuawilson) [inline src/app/treelist/treelist.component.ts]: I don't think this is needed now that we have it building the .css files.

Comment (dlabrecq) [inline src/app/treelist/treelist.component.ts]: Unfortunately, it is. I linked the latest with my PR from the planner UI. Without the toString, I still see the error message.

Comment (pmuir) [inline src/app/treelist/treelist.component.ts]: Almost certainly the loaders are misconfigured for .css files, as we normally use .scss. Please can you open an issue in planner and assign to me, I'll try to sort it out.

**Pull request merged**